website/source/docs/provisioners/file.html.markdown

```diff
@@ -49,13 +49,13 @@ resource "aws_instance" "web" {
 The following arguments are supported:
 
 * `source` - This is the source file or folder. It can be specified as relative
-  to the current working directory or as an absolute path. This cannot be provided with `content`.
+  to the current working directory or as an absolute path. This attribute cannot be specifed with `content`.
 
 * `content` - This is the content to copy on the destination. If destination is a file,
   the content will be written on that file, in case of a directory a file named
   *tf-file-content* is created. It's recommended to use a file as destination. A
-  [`template_file`](/docs/providers/template/r/file.html) might be referenced in here, or
-  any interpolation syntax for that matter. This cannot be provided with `source`.
+  [`template_file`](/docs/providers/template/index.html) might be referenced in here, or
```

Comment:
I think it would be better to link to the actual `template_file` data source - i.e. should be enough to just replace `/r/` with `/d/`.